### DISCUSSION TITLE
Gadgetbridge integration for smartwatches and other gadgets

### DISCUSSION THREAD

**Discussion by ghost** [iSoron/uhabits#1533] [Feature Requests]
Hi, I'd like to be able to get reminders of habits and tick them off using my smartwatch. 

Open source gadget solutions support Gadgetbridge: https://codeberg.org/Freeyourgadget/Gadgetbridge/

It would be great to have Habits integrate with Gadgetbridge. Hopefully this would also encourage people to buy open-source and non-invasive smartwear.

**Comment by iSoron** [comment]
> I'd like to be able to get reminders of habits and tick them off using my smartwatch.

Thanks for the suggestion, @andreasmjg. Our notifications already have actions (e.g. yes, no, later). On Android Wear, this was already enough for them to be actionable on the watch itself. Do you know what would be necessary to make them work in Gadgetbridge?
